libs/components/theme/src/lib/styles/_buttons.scss

```diff
@@ -148,6 +148,7 @@ a.sky-btn {
   font-weight: 400;
   color: var(--sky-text-color-default);
   line-height: 1.6em;
+  --sky-override-tab-line-height: 1.6em !important;
```

Comment:
Sorry if I'm missing something - why are we doing this here instead of in a default override in a component somewhere? We are doing the modern version in the component.

Reply:
The tabs have a different line height in gemini vs modern and in tabs vs wizard, so modern always has an override. This was the bare minimum to keep modern wizard buttons working before we tokenize them in their own story. I think specificity changed when removing the styles from the mixin, because this no longer takes effect over the line height set for regular tabs without the `!important`. I can find a cleaner approach if you'd like.

Reply:
It just felt odd to have an override variable just hanging out here outside of an override section. I'd like to clean that up if possible - but I won't die on that hill.